nss-ivory-trolls/WorkforceManagement
# Jb begin dept

# Description

## TICKET-5
Given user wants to view departments
When user clicks on the Departments section in the navigation bar
Then all current departments should be listed
And the following information should be presented to the user

Department name
Department budget
Size of department (number of employees assigned)


## Type of change


# Testing Instructions for Change Made

1. `git fetch --all`
1.  `git checkout jb-beginDept`
1.  open file with visual studio
1.   be sure that `appsettings.json` is pointing to `BangazonAPIWorkforceManagement` database
1.  in visual studio run server/mvc
1.  upon browser view click on nav bar option Departments
1.  in Departments window you should see various department names, budgets, and number of employees.


# Checklist:


🌮
# Description 
## TICKET-6 

Given user is viewing all departments
When user clicks on the Create New link
Then a form should be presented in which the new department name can be entered


## Type of change


# Testing Instructions for Change Made

1. `git fetch --all`
1.  `git checkout jb-beginDept`
1.  open file with visual studio
1.   be sure that `appsettings.json` is pointing to `BangazonAPIWorkforceManagement` database
1.  in visual studio run server/mvc
1.  upon browser view click on nav bar option Departments
1.  in Departments window you can click create to be taken to a new window in which you can create a new department .


# Checklist:
